plugin/src/main/kotlin/org/jetbrains/research/pynose/plugin/inspections/TestRunnerServiceFacade.kt

```diff
@@ -0,0 +1,23 @@
+package org.jetbrains.research.pynose.plugin.inspections
+
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.module.ModuleUtilCore
+import com.intellij.psi.PsiFile
+import com.jetbrains.python.testing.TestRunnerService
+
+@Service
+object TestRunnerServiceFacade {
+
+    private val moduleTestRunnerMap: MutableMap<Module, String> = mutableMapOf()
+
+    fun getConfiguredTestRunner(file: PsiFile): String? {
+        val module = ModuleUtilCore.findModuleForPsiElement(file) ?: return ""
```

Comment:
We need to cache Module by File here, not TestRunner by Module - I'm afraid that `findModule` call is time-consuming here.

Reply:
oh, yes, of course. idk what was going on in my head when i wrote that sorry haha